helpdesk_mgmt_maintenance/models/maintenance_equipment.py

```diff
@@ -0,0 +1,35 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields, api
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    ticket_ids = fields.Many2many('helpdesk.ticket', string = "Tickets")
+
+    allow_ticket = fields.Boolean(string="Allow Tickets", default=False)
+
+    num_tickets = fields.Integer(string="Tickets", compute="_compute_num_tickets", store=True)
```

Comment:
Please rename field to a more Odoo standard conventions one:
```suggestion
    ticket_count = fields.Integer(string="Ticket count", compute="_compute_ticket_count", store=True)
```